public/app/features/variables/query/reducer.test.ts

```diff
@@ -1,23 +1,23 @@
 import { reducerTester } from '../../../../test/core/redux/reducerTester';
 import { queryVariableReducer, updateVariableOptions, updateVariableTags } from './reducer';
-import { QueryVariableModel, VariableOption } from '../../templating/types';
+import { QueryVariableModel } from '../../templating/types';
 import cloneDeep from 'lodash/cloneDeep';
 import { VariablesState } from '../state/variablesReducer';
 import { getVariableTestContext } from '../state/helpers';
 import { toVariablePayload } from '../state/types';
 import { createQueryVariableAdapter } from './adapter';
+import { MetricFindValue } from '@grafana/data';
 
 describe('queryVariableReducer', () => {
   const adapter = createQueryVariableAdapter();
 
   describe('when updateVariableOptions is dispatched and includeAll is true', () => {
     it('then state should be correct', () => {
       const { initialState } = getVariableTestContext(adapter, { includeAll: true });
```

Comment:
nit: could we try to use the builder here instead of getVariableTestContext? I might be way off here just wanted to check if it's possible :)

Reply:
![yes, we can](https://i.giphy.com/media/l0ErF2i4YCrQwRBMQ/giphy.webp)